integrations/jira/.port/resources/blueprints.json

```diff
@@ -0,0 +1,87 @@
+[
+  {
+    "identifier": "project",
+    "title": "Jira Project",
+    "icon": "Jira",
+    "description": "A Jira project",
+    "schema": {
+      "properties": {
+        "url": {
+          "title": "Project URL",
+          "type": "string",
+          "format": "url",
+          "description": "URL to the project in Jira"
+        }
+      }
+    }
+  },
+  {
+    "identifier": "issue",
+    "title": "Jira Issue",
+    "icon": "Jira",
+    "schema": {
+      "properties": {
+        "url": {
+          "title": "Issue URL",
+          "type": "string",
+          "format": "url",
+          "description": "URL to the issue in Jira"
+        },
+        "status": {
+          "title": "Status",
+          "type": "string",
+          "description": "The status of the issue"
+        },
+        "issueType": {
+          "title": "Type",
+          "type": "string",
+          "description": "The type of the issue"
+        },
+        "components": {
+          "title": "Components",
+          "type": "array",
+          "description": "The components related to this issue"
+        },
+        "assignee": {
+          "title": "Assignee",
+          "type": "string",
+          "format": "user",
+          "description": "The user assigned to the issue"
+        },
+        "reporter": {
```

Comment:
Did you try using a higher level of a token?

If we can get the email i don't think there is a use for these fields